src/cascadia/WindowsTerminal/IslandWindow.h

```diff
@@ -8,6 +8,7 @@
 #include <winrt/Microsoft.Terminal.TerminalControl.h>
 #include <winrt/TerminalApp.h>
 #include "../../cascadia/inc/cppwinrt_utils.h"
+#include <windowsx.h>

```

Comment:
What specifically is using windowsx.h in this header? if it's only something in the cpp file, it should probably be in precomp or the cpp file.

Reply:
moved to pch.h